Casks/d/dropbox-capture.rb

```diff
@@ -12,20 +13,34 @@
   homepage "https://dropbox.com/capture/"
 
   livecheck do
-    url "https://dropbox.com/capture/download"
-    strategy :header_match
+    url "https://client.dropbox.com/squirrel/dropbox-capture/update_check/stable?localversion=0&arch=#{livecheck_arch}"
+    strategy :json do |json|
+      json["version"]
+    end
   end
 
+  deprecate! date: "2025-03-24", because: :discontinued
```

Comment:
```suggestion
  disable! date: "2025-03-24", because: :discontinued
```

I'm sorry for not catching this earlier. According to the article, after this date, downloading or accessing the app will no longer be possible. So we should disable the Cask.